### ISSUE TITLE
SPOC/SIL

### ISSUE THREAD

**Issue by stwalsh** [dwp/dwp-style-guide#25]
Do we need something on this? 
I want to use SIL in a work coach message but I don't know whether it's something to avoid. Doesn't even feel like it needs spelling out.

**Comment by JohnTurnbull99** [comment]
Yes - write out at first mention, as per usual guidance on acronyms.

Create a new entry in style guide for SIL. Say not to use SPOC.

**Comment by lolylena** [comment]
Proposed text:
### service innovation lead, SIL [UC]
Lower case unless you're talking about a specific person, for example "Brian Jones, Service Innovation Lead at Canterbury service centre”. The first time you use it on a page explain it in full, then refer to it by initials. Never use ‘SPOC’.

### SPOC [UC]
Do not use. Use ‘service innovation lead (SIL)' instead.
